eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -0,0 +1,102 @@
+parameters: 
+- name: ShouldPublish
+  type: boolean
+  default: false
+- name: BuildToolsPath
+  type: string
+  default: '$(Build.SourcesDirectory)/azure-sdk-build-tools'
+- name: NugetVersion
+  type: string
+  default: '5.4.x'
+- name: PartnerDropsSubscription
+  type: string
+  default: 'azuresdkpartnerdrops-kv Secrets'
+- name: PartnerDropsStorageName
+  type: string
+  default: 'azuresdkpartnerdrops'
+- name: PartnerDropsContainerName
+  type: string
+  default: 'mgmt-netsdk-signed'
+- name: PartnerDropsBlobDestPrefix
+  type: string
+  default: '$(System.DefinitionId)-$(Build.BuildId)-$(System.JobAttempt)'
+- name: DependsOn
+  type: string
+  default: Build
+
+stages:
+  - stage: Release
+    dependsOn: ${{parameters.DependsOn}}
+    jobs:
+      - deployment: Release
+        environment: nuget
+        pool:
+          name: azsdk-pool-mms-win-2019-general
+          vmImage: MMS2019
+
+        strategy:
+          runOnce:
+            deploy:
+              steps:
+                - checkout: self
+                - checkout: azure-sdk-build-tools
+
+                - download: current
+                  artifact: packages
+                  displayName: Download published Nuget Packages
+
+                - task: Powershell@2
```

Comment:
@chidozieononiwu should this just use https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/pipelines/templates/steps/create-tags-and-git-release.yml?

Reply:
After looking through the code as you suggested, the major difference is the eng/common version assumes that only one package is being operated on per time. This works because of the way each artifact is listed in the ci.yml that originates the build. the release stage then looks through and runs the script for each artifact. In the mgmt case it passes $(scope) to the build which could be a service directory with many packages. The older version of the script does the looping within. I think there are other minor differences also.

Reply:
OK that means we need to keep this script in the internal build tools. Still might be worth filing an issue to follow-up on consolidating those two.